crates/sui-indexer/src/models/transactions.rs

```diff
@@ -128,131 +123,7 @@ impl StoredTransaction {
-        } else {
-            None
-        };
-
-        let object_changes = if options.show_object_changes {
-            let object_changes = {
-                self.object_changes.into_iter().map(|object_change| {
-                        match object_change {
-                            Some(object_change) => {
-                                let object_change: IndexedObjectChange = bcs::from_bytes(&object_change)
-                                    .map_err(|e| IndexerError::PersistentStorageDataCorruptionError(
-                                        format!("Can't convert object_change bytes into IndexedObjectChange. tx_digest={:?} Error: {e}", tx_digest)
-                                    ))?;
-                                Ok(ObjectChange::from(object_change))
-                            }
-                            None => Err(IndexerError::PersistentStorageDataCorruptionError(format!("object_change should not be null, tx_digest={:?}", tx_digest))),
-                        }
-                    }).collect::<Result<Vec<ObjectChange>, IndexerError>>()?
-            };
-            Some(object_changes)
-        } else {
-            None
-        };
-
-        let balance_changes = if options.show_balance_changes {
-            let balance_changes = {
-                self.balance_changes.into_iter().map(|balance_change| {
-                        match balance_change {
-                            Some(balance_change) => {
-                                let balance_change: BalanceChange = bcs::from_bytes(&balance_change)
-                                    .map_err(|e| IndexerError::PersistentStorageDataCorruptionError(
-                                        format!("Can't convert balance_change bytes into BalanceChange. tx_digest={:?} Error: {e}", tx_digest)
-                                    ))?;
-                                Ok(balance_change)
-                            }
-                            None => Err(IndexerError::PersistentStorageDataCorruptionError(format!("object_change should not be null, tx_digest={:?}", tx_digest))),
-                        }
-                    }).collect::<Result<Vec<BalanceChange>, IndexerError>>()?
-            };
-            Some(balance_changes)
-        } else {
-            None
-        };
-
-        Ok(SuiTransactionBlockResponse {
-            digest: tx_digest,
-            transaction,
-            raw_transaction,
-            effects,
-            events,
-            object_changes,
-            balance_changes,
-            timestamp_ms: Some(self.timestamp_ms as u64),
-            checkpoint: Some(self.checkpoint_sequence_number as u64),
-            confirmed_local_execution: None,
-            errors: vec![],
-            raw_effects: self.raw_effects,
-        })
-    }
-    fn try_into_sender_signed_data(&self) -> IndexerResult<SenderSignedData> {
+    pub(crate) fn try_into_sender_signed_data(&self) -> IndexerResult<SenderSignedData> {
```

Comment:
Typically `into_` functions take `self` as parameter. Here I think it's better called `try_get...`